analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -54,12 +54,13 @@ typealias ManagedProjectFiles = Map<PackageManagerFactory, List<File>>
 typealias ProjectResults = Map<File, List<ProjectAnalyzerResult>>
 
 /**
- * A class representing a package manager that handles software dependencies. The package manager is referred to by its
- * [managerName]. The analysis of any projects and their dependencies starts in the [analysisRoot] directory using the
+ * A class to represent a package manager of the given [managerName] that handles dependencies for the given
```

Comment:
Q: For what purpose do we now use the manager name?

Reply:
A: For referring to it from configuration, e.g. to enable / disable it from the CLI.

Reply:
Thanks!